### ISSUE TITLE
Implement new core brand ingredients

### ISSUE THREAD

**Issue by shawnborton** [Expensify/App#12251]
Note: This issue is part of the greater **[Implement Our New Branding in NewDot](https://github.com/Expensify/Expensify/issues/239140)** project. 

Design Doc: https://docs.google.com/document/d/197fXHLlHM_RJA4-3MITwdu0N5e0HX5Q74sA4W80YMD8/edit#

**What is it:**
Updating our old colors, logos, icons, and fonts with new brand ingredients. Updating our old button styles to look like pill shapes, and updating our text input styles to use a single bottom-border style.

We've also decided to make the dark colors the new default colorway. 

![image](https://user-images.githubusercontent.com/2319350/199368024-70c88379-4973-4124-8ae2-141b2b90a27f.png)

 
**Does it require a mini design doc:**
Yes

**Design finished:**
Yes

cc @GabiHExpensify

**Comment by Luke9389** [comment]
We want [this design doc](https://docs.google.com/document/d/197fXHLlHM_RJA4-3MITwdu0N5e0HX5Q74sA4W80YMD8/edit#) to be reviewed ASAP because it's a fast change that dramatically improves our app. Also it's been lumped into WAQ (WhatsApp Quality) so the level of urgency is high (n7 to be specific).

**Comment by roryabraham** [comment]
I have read and reviewed this Design Doc!

**Comment by trentpetty** [comment]
I have read and reviewed this Design Doc!

**Comment by dangrous** [comment]
I have read and reviewed this Design Doc!

**Comment by nathanmetcalf** [comment]
I have read and reviewed this Design Doc!

**Comment by mountiny** [comment]
I have read and reviewed this Design Doc!

**Comment by StefStavri** [comment]
I have read and reviewed this design doc

**Comment by trjExpensify** [comment]
I have read and reviewed this Design Doc!

**Comment by JmillsExpensify** [comment]
I have read and reviewed this Design Doc!

**Comment by shannonmccallexfy** [comment]
I have read and reviewed this design doc

**Comment by RobertLadue** [comment]
I have read and reviewed this Design Doc!

**Comment by ryanschaffer** [comment]
Whoops, I reviewed this 2 weeks ago and didn't comment.

**Comment by ryanschaffer** [comment]
I have read and reviewed this Design Doc!

**Comment by anmurali** [comment]
I have read and reviewed this Design Doc!

**Comment by grgia** [comment]
Still in progress! Right now we're fixing the last few dark-mode/font bugs and TestRail QA for Applause

**Comment by JmillsExpensify** [comment]
Out of curiosity, why are we keeping this main issue open if the initial implementation is now complete? I think it's great that we already have a bunch of follow-on issues to track bugs, I just thought that everything from the initial push was already done. Is that right, and if so, how about closing this issue?

**Comment by grgia** [comment]
I think because we're still waiting on [text inputs](https://github.com/Expensify/App/pull/12447) to get merged

**Comment by JmillsExpensify** [comment]
Gotcha, thanks for pulling out that PR. Hopefully we can close the loop on that one when everyone returns next week, and at that point, close this individual issue out.

**Comment by Luke9389** [comment]
I'm iterating on reviews for Text Inputs daily. Hoping to see it merged soon!

**Comment by shawnborton** [comment]
I think we can close this one out since the text inputs have been merged.
